cookbooks/portal-analytics/src/components/EquinorLoader.tsx

```diff
@@ -0,0 +1,27 @@
+import type React from 'react';
+import { StarProgress } from '@equinor/fusion-react-progress-indicator';
+import styled from 'styled-components';
+
+const Styled = {
+  Wrapper: styled.div`
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100vw;
+    height: 100vh;
+    overflow: hidden;
+  `,
+};
+
+export const EquinorLoader = ({
```

Comment:
what is the intent of this component, does it load Equinor?

Reply:
it's copy pasted from dev-portal.